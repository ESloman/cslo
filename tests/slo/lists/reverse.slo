var x = [1, 2, 3, 4];
println(x);
println(x.reverse());

var y = [1];
println(y);
println(y.reverse());

var z = [];
println(z);
println(z.reverse());
