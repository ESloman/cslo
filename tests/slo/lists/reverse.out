list[4]: [1, 2, 3, 4]
list[4]: [4, 3, 2, 1]
list[1]: [1]
list[1]: [1]
list[0]: []
list[0]: []
